### ISSUE TITLE
Node type label should be kebab-cased when output as part of a CSS class

### ISSUE THREAD

**Issue by danwill** [verbb/navigation#272]
**Description**
This is a very minor issue of a very specific edge case, but I figured I would log it because the fix seems simple enough.

When using a custom [Node Type](https://verbb.io/craft-plugins/navigation/docs/developers/extending-elements#node-types) where the node type has a displayName property that is more than one word, any CSS class names that use that displayName property have the potential to break layout since displayName will retain the spaces instead of converting them to kebab-case.

In my particular situation, I created a node type of "Mega Menu" and noticed that its type label wasn't appearing in the build view. Turns out, the class applied to the label was `node-type-mega menu` and the dangling `.menu` class was setting the element to `display: none`.

Relevant plugin code is in `verbb/navigation/src/templates/navs/_build.html`, line 245:
```
<span class="node-type-{{ element.getNodeTypeLabel() }}" title="{{ element.url }}">{{ element.getNodeTypeLabel() }}</span>
```

**Steps to reproduce**
1. Create a custom [Node Type](https://verbb.io/craft-plugins/navigation/docs/developers/extending-elements#node-types) where the node type has a returned displayName that is more than one word, and the second word is "Menu".
2. Add that custom type to a navigation and see that the type label doesn't appear.

**Additional info**
- Plugin version:
- Craft version: 3.7.20
- Multi-site: No
- Propagate nodes: No

**Comment by engram-design** [comment]
Yep, great pick up! Should be fixed for the next release. To get the fix early, change your `verbb/navigation` requirement in `composer.json` to:

```
"require": {
  "verbb/navigation": "dev-craft-3 as 1.4.22",
  "...": "..."
}
```

Then run `composer update`.

**Comment by danwill** [comment]
Thanks for the quick turnaround. I updated to the next release and can confirm that the fix is working!

**Comment by engram-design** [comment]
Fixed in [1.4.23](https://github.com/verbb/navigation/releases/tag/1.4.23)
